src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs

```diff
@@ -24,17 +25,23 @@ public class ImageShortcodeProvider : IShortcodeProvider
         private readonly IHtmlSanitizerService _htmlSanitizerService;
         private readonly IHttpContextAccessor _httpContextAccessor;
         private readonly ResourceManagementOptions _options;
+        private readonly IFileVersionProvider _fileVersionProvider;
+        private readonly IOrchardHelper _orchardHelper;
 
         public ImageShortcodeProvider(
             IMediaFileStore mediaFileStore,
             IHtmlSanitizerService htmlSanitizerService,
             IHttpContextAccessor httpContextAccessor,
-            IOptions<ResourceManagementOptions> options)
+            IOptions<ResourceManagementOptions> options,
+            IFileVersionProvider fileVersionProvider,
+            IOrchardHelper orchardHelper)
```

Comment:
Why are you suing `IOrchardHelper`? This should not be used here. Instead you can use `_httpContextAccessor.HttpContext` to do what you need,